Assets/MaxPecora/Project 2/Advanced Controller/Scripts/PlayerController.cs

```diff
@@ -109,9 +110,13 @@ void Update()
 
             if (Input.GetKeyDown(KeyCode.R))
             {
-                Application.LoadLevel(0); //or whatever number your scene is
+                //do the following, specific named scene loading with SceneManager
+                string sceneName = SceneManager.GetActiveScene().name;
+                SceneManager.LoadScene(sceneName);
```

Comment:
Definitely take a couple minutes to understand this.  I didn't try to find your next level script so that part is likely still broken sadly in the final build, but we can take some time to fix it.